include/cute/util/debug.hpp

```diff
@@ -126,7 +126,12 @@ CUTE_HOST_DEVICE
 bool
 block(int bid)
 {
-#if defined(__CUDA_ARCH__)
+#if defined(CUTLASS_ENABLE_SYCL)
+    using namespace syclcompat;
+    return (work_group_id::x() +
+       work_group_id::y() * global_range::x() +
+       work_group_id::z() * global_range::x() * global_range::y() == bid);
```

Comment:
```suggestion
    using sycl::ext::oneapi::experimental::this_nd_item;
    return (this_nd_item<3>.get_linear_id()==bid);
```

Reply:
I think this will simplify the job as you wanted to calculate the linear_id here